internal/binutils/binutils.go

```diff
@@ -140,7 +144,65 @@ func initTools(b *binrep, config string) {
 		b.addr2line, b.addr2lineFound = findExe("gaddr2line", append(paths["addr2line"], defaultPath...))
 	}
 	b.nm, b.nmFound = findExe("nm", append(paths["nm"], defaultPath...))
-	b.objdump, b.objdumpFound = findExe("objdump", append(paths["objdump"], defaultPath...))
+	// Look for llvm-objdump and objdump.
```

Comment:
nit: I'd remove this comment. The doc to findObjdump function is sufficient. And this comment is incomplete since it doesn't mention gobjdump. It's better to keep the comment in one place, at findObjdump.

Reply:
Done.